src/components/cylc/gscan/GScan.vue

```diff
@@ -106,71 +106,62 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         v-if="!isLoading && sortedWorkflows && sortedWorkflows.length > 0"
         class="c-gscan-workflows"
       >
-        <div
```

Comment:
Above this part we have the GScan filter, recently merged :point_up: did some manual testing after fixing conflicts, but thought it was still good to let reviewers know what changed recently